docs/source/install/common/start_services.rst

```diff
@@ -2,6 +2,6 @@
 
     sudo st2ctl start
 
-* Register sensors and actions ::
+* Register sensors, rules and actions ::
```

Comment:
Sorry, you are right.

We updated `st2ctl reload` to also register rules by default (but not actual `st2-register-content` script which is called by st2ctl).

So, LGTM